google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -35,6 +35,7 @@ appengine.promote.label=Promote the deployed version to receive all traffic
 appengine.promote.info.label={0}To manually promote a version use the {1}Google Cloud Console{2}{3}
 appengine.stop.previous.version.label=Stop previous version
 appengine.environment.label=Environment\:
+appengine.cloudsdk.devappserver.missing=Selected Cloud SDK does not contain the app-engine-java component.
```

Comment:
Oh, it comes up in the UI?

In that case maybe we can split up the message I created in my other PR into 2 parts, and you consume only the first part of it. What do you think?